ubiquity-os/ubiquity-os-plugin-installer
# Fix/org choices

Resolves #26
Requires #25 

Changes:

1. `rest.users.getAuthenticated()`
2. `rest.orgs.listForAuthenticatedUser()`
3. `rest.orgs.listForUser(username)`
4. `rest.repos.getCollaboratorPermissionLevel(org, .ubiquity-os)`

QA:

![image](https://github.com/user-attachments/assets/57e1725f-9a05-423d-9e5d-fa9c4e971ae1)

Console:

![image](https://github.com/user-attachments/assets/dba19381-54cc-4110-b87c-27ec0d056ab3)

Else:

> (4) ['ubq-testing', 'ubiquity', 'ubiquibot', 'ubiquity-os']

I tried every `.rest.org` method that seemed relevant and none of them returned `-marketplace` for me so idk if that's a me problem or an org problem but it should be fetched at least and either errored in the console or displayed but it's neither. It also didn't display for you either @gentlementlegen and we are both public members of the org.

Could you try to see if `-marketplace` shows up for you now with this change?
